### PR TITLE
Setup project config for Docusaurus

### PR DESCRIPTION
**Pull request by hs-saunved**
This PR sets up the high-level project config for Docusaurus

### CODE REVIEW

File: src/data/footerLinks.json
Comment (helpshift-ps) [inline src/data/footerLinks.json:56]: This is to contact from a sales/company perspective. We should also have a "report a bug" or contact us for support link. https://www.helpshift.com/company/contact-us/ can probably be replaced to cover those functionalities instead.

File: src/data/variables.json
Comment (helpshift-ps) [inline src/data/variables.json:7]: The latest SDK X iOS version is 10.2.1. 
Will updating the version here be part of the SDK X dev docs release process? Please work with the mobile team to create a guide for SDK release dev docs changes.

File: src/data/variables.json
Comment (helpshift-ps) [inline src/data/variables.json:16]:
```suggestion
            "sdk_version": "v10.2.1",
```

File: src/css/custom.css
Comment (hs-saunved) [reply src/css/custom.css:16]: It's not needed, removing it

File: src/css/custom.css
Comment (hs-saunved) [reply src/css/custom.css:9]: Ack

File: src/css/custom.css
Comment (hs-saunved) [reply src/css/custom.css:15]: Ack, this has been picked up from the Docusaurus docs, I can link it here

File: src/data/footerLinks.json
Comment (hs-saunved) [reply src/data/footerLinks.json:56]: We will be showing the Web Chat widget on this page. Maybe the "Report a bug" button can open the chat widget? I can add this as a backlog task

File: src/data/variables.json
Comment (hs-saunved) [reply src/data/variables.json:7]: Yes. This is similar to updating the settings.json file in `quill`. I will be creating a document for how this will be updated for SDK X releases and sharing it soon.

File: src/data/variables.json
Comment (hs-saunved) [reply src/data/variables.json:12]: Not changing this right now. This will be done in a new PR where all the versions and URLs will be fixed

File: src/data/variables.json
Comment (hs-saunved) [reply src/data/variables.json:20]: This has been carried forward from the old settings.json in `quill`. I didn't want to change it for fear of breaking something I didn't know.  We can fix this in future PRs. Adding this to my backlog.

File: src/data/footerLinks.json
Comment (hs-saunved) [reply src/data/footerLinks.json:27]: Yeah, makes sense. Will remove this.

**Review by nrkin — COMMENTED** [review]
LGTM

File: src/data/footerLinks.json
Comment (helpshift-ps) [reply src/data/footerLinks.json:52]: Sure, please create a backlog and resolve.

File: src/data/variables.json
Comment (helpshift-ps) [reply src/data/variables.json:7]: Okay. Please create a backlog and resolve this comment.

**Review by helpshift-ps — APPROVED** [review]

File: src/data/footerLinks.json
Comment (hs-saunved) [reply src/data/footerLinks.json:52]: Added backlog task

File: src/data/variables.json
Comment (hs-saunved) [reply src/data/variables.json:7]: Added backlog task

**Comment by hs-saunved** [comment]
Ready to merge (after furthest branch is merged)

**Pull request merged**
